build.gradle

```diff
@@ -74,6 +74,11 @@ subprojects {
         withJavadocJar()
         withSourcesJar()
     }
+    configurations.runtimeClasspath {
```

Comment:
Could we add some comments here to clarify how is affecting the deps validation? e.g. usage of platform feature, etc.